gapis/server/server.go

```diff
@@ -303,6 +303,20 @@ func (s *server) SplitCapture(ctx context.Context, rng *path.Commands) (*path.Ca
 	}
 }
 
+func (s *server) TrimCaptureInitialState(ctx context.Context, p *path.Capture) (*path.Capture, error) {
+	ctx = log.Enter(ctx, "TrimCaptureInitialState")
+	c, err := capture.ResolveGraphicsFromPath(ctx, p)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, api := range c.APIs {
+		c.InitialState.APIs[api].TrimInitialState(ctx, p)
+	}
```

Comment:
This appears to modify the capture. I think it would be better if we cloned the initial state, then trimmed the cloned state, and created a new capture with the trimmed state. Especially since that's what the comment in the service.proto says :)

Reply:
indeed! done